File: ParkingObserverC#/ParkingObserver/Observer/Carros.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace ParkingObserver.Observer
{
    public class Carros : ICarros //class Investor
    {
        public string Nome { get; set; }
        public int Andar { get; set; }

        //Construtor
        public Carros(string nome, int andar)
        {
            this.Nome = nome;
            this.Andar = andar;
        }

        public void Update(Estacionamento estacionamento)
        {
            Console.WriteLine(estacionamento.Symbol + " " + Nome + " está no andar " + Andar + " - " + "Vagas liberada em: QuadraX, CorredorY");
        }

        //Gets or sets the Estacionamento
       // private Estacionamento Estacionamento { get; set; }

    }
}

File: ParkingObserverJava/src/br/Observer/Carros.java
package br.Observer;

public class Carros implements ICarros {
	
	public String Nome;
	public int Andar;
	
public int getAndar() {
		return Andar;
	}

	public void setAndar(int andar) {
		Andar = andar;
	}

public String getNome() {
		return Nome;
	}

	public void setNome(String nome) {
		Nome = nome;
	}

public Carros(String nome, int andar)
{
	this.Nome = nome;
	this.Andar = andar;
}

@Override
public void Update(Estacionamento estacionamento) {
   // Console.WriteLine(estacionamento.Symbol + " " + Nome + " está no andar " + Andar + " - " + "Vagas liberada em: QuadraX, CorredorY");
   // System.out.println(tv1.isLigado() ? "ligada" : "desligada");
}

}

